### PR TITLE
LP-2773 Reset Password Link redirects user to Page Not Found error 

### PR DESCRIPTION
**Pull request by annas-usman**
[Ticket ](https://philanthropyu.atlassian.net/browse/LP-2773)

### CODE REVIEW

**Comment by annas-usman** [comment]
The issues reported by the quality action are in the core code, hence they can not be fixed.

**Review by asadali145 — APPROVED** [review]

File: openedx/core/djangoapps/user_authn/views/login.py [hunk 1]
```diff
@@ -142,7 +142,7 @@ def _generate_locked_out_error_message():
                                      'locked. Try again in {locked_out_period} minutes.\n'
                                      'To be on the safe side, you can reset your '
                                      'password {link_start}here{link_end} before you try again.\n')).format(
-            link_start=HTML('<a href="/reset" >'),
+            link_start=HTML('<a http="/login" class="form-toggle" data-type="password-reset">'),
```
Comment (asadali145) [inline openedx/core/djangoapps/user_authn/views/login.py:145]: Please run the core unit tests for this app, I think tests will fail. Use `is_testing_env` flag for this change if tests are failing.

File: openedx/core/djangoapps/user_authn/views/login.py [hunk 2]
```diff
@@ -255,7 +255,7 @@ def _handle_failed_authentication(user, authenticated_user):
                                                  ))
                                           .format(
                         quote=HTML("'"),
-                        link_start=HTML('<a href="/reset" >'),
+                        link_start=HTML('<a http="/login" class="form-toggle" data-type="password-reset">'),
```
Comment (asadali145) [inline openedx/core/djangoapps/user_authn/views/login.py:258]: Same as mentioned above.

**Review by asadali145 — COMMENTED** [review]
Added a comment have a look pls.

File: openedx/core/djangoapps/user_authn/views/login.py (hunk 1 above)
Comment (annas-usman) [reply openedx/core/djangoapps/user_authn/views/login.py:145]: No tests are failing because of the changes

**Review by asadali145 — APPROVED** [review]

**Review by AliAdnanSohail — APPROVED** [review]

**Pull request merged**